### ISSUE TITLE
control numbering

### ISSUE THREAD

**Issue by xtimbeau** [ute/custom-numbered-blocks#4]
Hi great extension, quite neat.
It would be nice to have control over numbering. For instance flat numbering (all items are numbering continuously all along the document, could specifiy a start counter, could specify letter counter or roman numbers).
Groups and listing are really cool.

**Comment by ute** [comment]
yes, this could be nice, indeed, and is on my wish list. Quarto will get a new crossreferencing mechanism soonish, and this may determine how to implement the numbering features you suggest. Therefore I postponed it for myself, but if anyone would like to contribute here, I will happily merge PRs :-)

Just needed something that numbers by section when I wrote the extension, so flat numbering could be something I implement in the nearer future - number by section is a bit ridiculous for short documents.

**Comment by ute** [comment]
It should work now :-) 
You can chose if you number per chapter/section or not. Uses the same syntax and defaults as Quarto since its crossref overhaul (Quarto 1.4)
